addons/binding/org.openhab.binding.hyperion/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,184 @@
+			<channel id="smoothing" typeId="smoothing" />
+			<channel id="kodichecker" typeId="kodichecker" />
+			<channel id="forwarder" typeId="forwarder" />
+			<channel id="udplistener" typeId="udplistener" />
+			<channel id="boblightserver" typeId="boblightserver" />
+			<channel id="grabber" typeId="grabber" />
+			<channel id="v4l" typeId="v4l" />
+			<channel id="leddevice" typeId="leddevice" />
+		</channels>
+
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>Host address</label>
+				<context>network_address</context>
+				<description>The host address of the Hyperion server JSON API.</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="port" type="integer" required="true">
+				<label>Port</label>
+				<description>The port of the Hyperion server JSON API.</description>
+				<required>true</required>
+				<default>19444</default>
+			</parameter>
+			<parameter name="priority" type="integer" required="true">
+				<label>Priority</label>
+				<description>The priority associated with updates sent to the Hyperion server.</description>
+				<required>true</required>
+				<default>50</default>
+			</parameter>
+			<parameter name="poll_frequency" type="integer" required="true">
+				<label>Polling frequency</label>
+				<description>How often (in seconds) to poll the Hyperion server for value changes.</description>
+				<required>true</required>
+				<default>3</default>
+			</parameter>
+			<parameter name="origin" type="text" required="true">
+				<label>Origin</label>
+				<description>The origin used on color and effect commands.</description>
+				<required>true</required>
+				<default>openHAB</default>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="brightness">
+		<item-type>Dimmer</item-type>
+		<label>Brightness</label>
+		<description>Sets the brightness of the LEDs</description>
+	</channel-type>
+	<channel-type id="color">
+		<item-type>Color</item-type>
+		<label>Color</label>
+		<description>Sets the color effect of the LEDs</description>
+	</channel-type>
+
+	<channel-type id="effect">
+		<item-type>String</item-type>
+		<label>Effect</label>
+		<description>Sets the effect of the LEDs</description>
+	</channel-type>
+
+	<channel-type id="clear" advanced="true">
+		<item-type>String</item-type>
+		<label>Clear priority</label>
+		<description>Clears the given Hyperion priority</description>
+	</channel-type>
+
+	<channel-type id="enabled">
+		<item-type>Switch</item-type>
+		<label>Enabled</label>
+		<description>Shows whether Hyperion is currently enabled or not and allows enabling or disabling of Hyperion</description>
+	</channel-type>
+
+	<channel-type id="blackborder" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Black border</label>
+		<description>Shows the current state of the component and allows enabling or disabling the component</description>
```

Comment:
Please provide a more meaningful description for each of the "components".

Reply:
done